### DISCUSSION TITLE
Alerts on Frames: What do they mean

### DISCUSSION THREAD

**Discussion by crisedwards** [senshu/Sozi#562] [Q&A] [answered]
Just started using Sozi and am getting the "alert" signs on my frames. I can't figure out what these are alerting me to:

![Screen Shot 2021-10-02 at 8 57 52 PM](https://user-images.githubusercontent.com/491089/135736937-36232da0-22bf-4352-b66f-80a041b9fab1.png)

**Comment by aumouvantsillage** [comment, marked as answer]
If you put the mouse pointer over one of these warning signs, you should see the following message: "You should add graphic elements in the current area to help Sozi keep track of this layer's position."

It means that Sozi cannot find a graphic element that it could use as an "anchor" for the current frame and the current layer. This frame will be positioned using absolute coordinates and will not be repositioned automatically if you move things in your SVG.

**Reply by crisedwards** [reply]
Ahhhh! Interesting. My presentation is almost entirely text. I did mouseover the warnings and no message appeared, so I am glad I asked. I will experiment with graphic [non-text] elements to see if that works. Thanks for the input! I appreciate your help.
